server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -790,27 +790,24 @@ private ClusterMetadataManifest uploadManifest(
                 committed,
                 MANIFEST_CURRENT_CODEC_VERSION
             );
-            final ClusterMetadataManifest manifest = new ClusterMetadataManifest(
-                clusterState.term(),
-                clusterState.getVersion(),
-                clusterState.metadata().clusterUUID(),
-                clusterState.stateUUID(),
-                Version.CURRENT,
-                nodeId,
-                committed,
-                MANIFEST_CURRENT_CODEC_VERSION,
-                null,
```

Comment:
what was this null before?

Reply:
Yes, this null attribute is for global metadata file name. Which was removed after split PR.